app/views/layouts/_emails.html.erb

```diff
@@ -0,0 +1,5 @@
+<% if user.additional_email.blank? %>
+  <%= mail_to user.email, user.email %>
+<% else %>
+  <%= mail_to user.additional_email, user.additional_email %>
```

Comment:
Why not add some method for getting a public email to the `User` model instead?
It would a single line like
```
<%= mail_to user.public_email, user.public_email %>
```
in both cases instead of using additionallayout.